index.html

```diff
@@ -135,15 +135,16 @@ <h1>About me</h1>
   </li>
 </ul>
 </section>
-<footer class="aboutContact" id="contact">
+<footer class="aboutContact" >
   <div class="formResponsive">
   <p>
     I'm alwyas ineterested in hearing about new projects, so if you'd 
     like to chat please get in touch.
   </p>
-  <form action="https://formspree.io/f/mvodrqbw" method="POST" class="contactForm">
-    <input type="text" name="user_name" placeholder="Full Name" maxlength="30" required>
-    <input type="email" name="user_email" placeholder="Email address" required>
+  <form action="https://formspree.io/f/mvodrqbw" method="POST" class="contactForm" id="contact">
+    <input type="text" name="user_name" id="username" placeholder="Full Name" maxlength="30" required>
+    <input type="email" name="user_email" id="email" placeholder="Email address">
```

Comment:
- [ ] The email error code should be near the submit button.

Reply:
![Screenshot_20210701-154100_WhatsApp](https://user-images.githubusercontent.com/58504655/124111600-e6570b80-da82-11eb-9a54-6848436b1e94.jpg)